tools/core-build-tasks/src/tasks/helpers/copyFiles.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mojang AB.  All rights reserved.
+
+import { FileSystem } from '@rushstack/node-core-library';
+import path from 'path';
+
+export function copyFiles(originPaths: string[], outputPath: string) {
+    const destinationPath = path.resolve(outputPath);
+    for (const originPath of originPaths) {
+        const inputPath = path.resolve(originPath);
+        const pathStats = FileSystem.getLinkStatistics(inputPath);
+        if (pathStats.isDirectory()) {
+            console.log(`Copying folder ${inputPath} to ${destinationPath}`);
+            FileSystem.copyFiles({
+                sourcePath: inputPath,
+                destinationPath: destinationPath,
+            });
+        } else {
+            const filename = path.parse(inputPath).base;
+            const destinationFilePath = path.resolve(destinationPath, filename);
+            console.log(`Copying file ${inputPath} to ${destinationPath}`);
+            FileSystem.copyFiles({
+                sourcePath: inputPath,
+                destinationPath: destinationFilePath,
+            });
+        }
```

Comment:
nit, could probably be simplified to
```suggestion
        if (pathStats.isDirectory()) {
            console.log(`Copying folder ${inputPath} to ${destinationPath}`);

        } else {
            const filename = path.parse(inputPath).base;
            destinationFilePath = path.resolve(destinationPath, filename);
            console.log(`Copying file ${inputPath} to ${destinationPath}`);
        }
        FileSystem.copyFiles({
            sourcePath: inputPath,
            destinationPath: destinationPath,
         });
```

just make sure to change the declaration of destinationPath to a let

Reply:
Ok, refactored